Assets/GothicVR/Scenes/Bootstrap.unity

```diff
@@ -1614,6 +1616,7 @@ GameObject:
   - component: {fileID: 1896514223}
   - component: {fileID: 1896514226}
   - component: {fileID: 1896514228}
```

Comment:
Please change _SkyManager.cs_ to _static_. You can then subscribe to GeneralSceneLoaded to load the actual sky. (Where you can also check for world name to load only for world.zen).

Background: We want to have as few SingletonBehaviours in our Bootstrap.unity scene as possible. As it is complex to read and adds potential merge conflicts and renaming issues in the future.

Reply:
I would rather not have SkyManager as a static class as it's not used as a helper class to provide only functions to be used, but rather having the sky state available for interpolation